### PR TITLE
testing

### PR DESCRIPTION
**Pull request by whosFritz**
testing

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 FROM openjdk:17-jdk-slim
 COPY target/*.jar app.jar
-EXPOSE 8080
+EXPOSE 8654
```
Comment (olech2412) [inline Dockerfile:3]: Please explain

File: compose.yaml [hunk 2]
```diff
@@ -2,9 +2,13 @@ services:
   mariadb:
     image: 'mariadb:latest'
     environment:
-      - 'MARIADB_DATABASE=mydatabase'
-      - 'MARIADB_PASSWORD=secret'
       - 'MARIADB_ROOT_PASSWORD=verysecret'
-      - 'MARIADB_USER=myuser'
+      - 'MARIADB_DATABASE=railmetrics'
+      - 'MARIADB_PASSWORD=password'
+      - 'MARIADB_USER=user'
```
Comment (olech2412) [inline compose.yaml:8]: for development maybe root?

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
-spring.datasource.url=jdbc:h2:mem:testdb
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=sa
+spring.datasource.url=jdbc:mariadb://localhost:3306/railmetrics
+spring.datasource.username=user
```
Comment (olech2412) [inline src/main/resources/application.properties:2]: same, but i edited the application.properties in 5-datenabruf-mit-async-und-scheduled-einrichten so ok for the moment

**Review by olech2412 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (whosFritz) [reply Dockerfile:3]: random port

File: compose.yaml (hunk 2 above)
Comment (whosFritz) [reply compose.yaml:8]: ok

File: src/main/resources/application.properties (hunk 3 above)
Comment (whosFritz) [reply src/main/resources/application.properties:2]: spring.datasource.url=jdbc:mariadb://localhost:3306/railmetrics
spring.datasource.username=user
spring.datasource.password=password
spring.datasource.driver-class-name=org.mariadb.jdbc.Driver
spring.jpa.hibernate.ddl-auto=update

this is the newest version, i dk why its in the old one

**Review by olech2412 — APPROVED** [review]

**Pull request merged**